### DISCUSSION TITLE
How to get label and value both from a ProFormSelect?

### DISCUSSION THREAD

**Discussion by KrishnaPG** [ant-design/pro-components#2764] [Q&A]
In a ProForm when ProFormSelect component is used, `form.getFieldValue()` returns only the `value` property of the select box. How to get the `label` also ( in addition to the `value`)?

After the form is submitted, would like to show a summary page with the selected option value and label both. But from the form there does not seem to be anyway to get the selected label. Any help is highly appreciated. Thank you.

**Comment by datnguyen3003** [comment]
You can use 'labelInValue' props of select component then enjoy it 👍
